### PR TITLE
use Nullable from androidx

### PR DESCRIPTION
**Pull request by Copser**
- use nullable import from maven to support androidx

### CODE REVIEW

File: android/src/main/java/com/vonovak/AddCalendarEventModule.java [hunk 1]
```diff
@@ -11,9 +11,11 @@
 import android.provider.CalendarContract;
 import android.app.LoaderManager;
 import android.content.Loader;
-import android.support.annotation.Nullable;
 import android.util.Log;
 
+// support for AndroidX
+import javax.annotation.Nullable;
```
Comment (vonovak) [inline android/src/main/java/com/vonovak/AddCalendarEventModule.java]: ah, please use the android X nullable here, not javax, thanks!

Comment (Copser) [reply android/src/main/java/com/vonovak/AddCalendarEventModule.java]: @vonovak no problem, check new changes thanks

**Comment by Copser** [comment]
hey, @vonovak is all good here? :)

**Comment by vonovak** [comment]
Thanks!

**Comment by Ashutosh-Tiwari** [comment]
@vonovak I see that the PR is merged but I am still seeing this error even if using the latest version of this plugin:

image

image

**Comment by laura-delucca-deliverit** [comment]
> @vonovak I see that the PR is merged but I am still seeing this error even if using the latest version of this plugin:
> 

me too

**Pull request merged**